lib/galaxy/web_stack/message.py

```diff
@@ -47,7 +47,7 @@ def dispatch(self, msg_str):
 
 
 class ApplicationStackMessage(dict):
-    _target: Optional[str] = None
+    target: Optional[str] = None  # type: ignore
```

Comment:
What is ignored here? It has a type annotation.

Reply:
It conflicts with the ``@property`` defined below named ``target``. I tried to rename to the class attribute so it doesn't conflict with the ``@property`` but it broke the json encoding.